2025-03-04 03:55:37.929 [info] 




2025-03-04 03:55:37.948 [info] Extension host agent started.
2025-03-04 03:55:38.056 [info] [<unknown>][fb331a27][ExtensionHostConnection] New connection established.
2025-03-04 03:55:38.070 [info] [<unknown>][9e7ce0ca][ManagementConnection] New connection established.
2025-03-04 03:55:38.182 [info] [<unknown>][fb331a27][ExtensionHostConnection] <210142> Launched Extension Host Process.
2025-03-04 03:55:41.482 [error] CodeExpectedError: Could not find pty 5 on pty host
    at I.W (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:12233)
    at I.updateTitle (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:7722)
    at O.s.<computed> (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:45:2962)
    at Object.call (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:28:4204)
    at Kc.s (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:80170)
    at Kc.q (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79693)
    at hs.value (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:79095)
    at C.B (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2373)
    at C.fire (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:26:2591)
    at process.w (file:///root/.vscode-server/cli/servers/Stable-e54c774e0add60467559eb0d1e229c6452cf8447/server/out/vs/platform/terminal/node/ptyHostMain.js:24:29136)
    at process.emit (node:events:518:28)
    at emit (node:internal/child_process:950:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-03-04 03:55:56.260 [error] [File Watcher] Unexpected error: inotify_add_watch on '/root/.vscode-server/data/logs/20250221T012354' failed: No such file or directory (EUNKNOWN) (path: /root)
2025-03-04 03:55:56.272 [error] [File Watcher (universal)] inotify_add_watch on '/root/.vscode-server/data/logs/20250221T012354' failed: No such file or directory
